pkg/providers/instancetype/suite_test.go

```diff
@@ -146,7 +146,37 @@ var _ = Describe("InstanceType Provider", func() {
 		ExpectCleanedUp(ctx, env.Client)
 	})
 
-	Context("subscription level quota error responses", func() {
+	Context("vm creation error responses", func() {
+
+		It("should delete the network interface on failure to create the vm", func() {
+			genericAzureErrMsg := "some generic error message"
```

Comment:
nit: i personally like this fun variable naming, but maybe better to make it formal. 
```
errorMsg := "test error"
errorCode := "400"

Reply:
it was originally "genericDepressedAzureErrMsg" but I see i need to formalize it a bit more :)

Reply:
fixed.